### DISCUSSION TITLE
Release Build

### DISCUSSION THREAD

**Discussion by paramThakkar1505** [PostgREST/postgrest#3579] [Q&A] [answered]
Is the build provided under the pre-builds a Release build? When I am trying to make it as windows service it is not allowing to do so. So Can I have release build of this?

**Comment by wolfgangwalther** [comment]
> Is the build provided under the pre-builds a Release build?

What do you mean by "pre-builds"?

What do you mean by "Release build"?

> When I am trying to make it as windows service it is not allowing to do so. 

It's unclear which problem you are facing here.

**Reply by paramThakkar1505** [reply]
I am using the exe downloaded from this link "https://github.com/PostgREST/postgrest/releases/latest" which is called as pre-builts in postgrest documentation. Now I am trying to create a windows background service using this EXE but while starting the windows background service it is throwing me below error.

![image](https://github.com/PostgREST/postgrest/assets/90238286/b747945b-9782-4bac-a46d-40d6f2b6b914)

Is it possible to create windows service using the postgrest.exe?

**Reply by wolfgangwalther** [reply, marked as answer]
If using the artifact from the release page, you need to provide the proper libraries / dependencies, too. The artifact alone won't work.

On Windows, it's much easier to use any of the package managers to install it. Choclatey / scoop or so? Not sure about that.

**Reply by wolfgangwalther** [reply]
That being said, the documentation really lacks some key information on that install page, agreed.

**Reply by paramThakkar1505** [reply]
Yes. Are you going to update the documentation?

**Reply by wolfgangwalther** [reply]
I don't plan to do it myself right now. Feel free to open PR, if you like.
